src/main/java/duke/gui/Main.java

```diff
@@ -0,0 +1,53 @@
+package duke.gui;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import duke.Duke;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for duke.Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Duke duke = new Duke();
+
+    @Override
+    public void start(Stage stage) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(Main.class.getResource("/view/MainWindow.fxml"));
+            AnchorPane ap = fxmlLoader.load();
+            fxmlLoader.<MainWindow>getController().setDuke(duke);
+            Scene scene = new Scene(ap);
+            stage.setTitle("EXE ChatBot (\"\\('^-^')/\")");
+            stage.setScene(scene);
+            stage.show();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
In general, printing stack trace is not good error handling. Think if there is anything better you can do about the exception rather than printing the stack trace.